examples/bootstrap/docusaurus.config.js

```diff
@@ -1,4 +1,4 @@
-/** @type {import('@docusaurus/types').DocusaurusConfig} */
+/** @type {import('@docusaurus/core/lib').DocusaurusConfig} */
```

Comment:
is the problem due to not having @docusaurus/types in deps?

Reply:
Yeah, so resolving `@docusaurus/types` will fail in e.g. yarn2 PnP, and it *may* fail with a `node_modules` directory depending on how the package manager chooses to set it up.  The package manager isn't required to hoist in a particular way -- or at all -- only to respect each `package.json`'s declared dependencies.